app/uk/gov/hmrc/customs/declaration/controllers/CustomHeaderNames.scala

```diff
@@ -24,4 +24,5 @@ object CustomHeaderNames {
   val NonRepudiationReceiptId: String = "X-Receipt-ID"
```

Comment:
please updated `README.md` with mention of the new api-subscription-field and include it in the the example curl statement `curl -v -X PUT "http://localhost:9650/field/application/d65f2252-9fcf-4f04-9445-5971021226bb/context/customs%2Fdeclarations/version/2.0" -H "Cache-Control: no-cache" -H "Content-Type: application/json" -d '{ "fields" : { "callbackUrl" : "https://postman-echo.com/post", "securityToken" : "securityToken" } }'`